docs/source/installation/for_dev.rst

```diff
@@ -69,6 +69,9 @@ installation with Poetry.
 Installing Manim using Poetry
 *****************************
 
+#.  First start by cloning the `manim repo <https://github.com/manimcommunity/manim>`_ and moving into
+    the folder you just cloned.
```

Comment:
I believe it's better to write the instructions in terms of `git clone` and `cd manim`, etc., in order to be more beginner-friendly. What do you think ?

Reply:
I've changed this via ee7b3ee08f692ccad7a3c7eaecf22fa0aea4ae3f, it now links to the document where we already explain working with git.